app/assets/javascripts/3dbio_viewer/src/webapp/components/protvista/ProtvistaPdbValidation.tsx

```diff
@@ -182,22 +198,190 @@ function useGrid() {
+            .attr("width", dimensions.width * 2)
+            .attr("height", dimensions.height * 2);
+
+        const svgBar = svg.append("g")
+
+        const bar = svgBar
+            .append("rect")
+            .attr("width", dimensions.width)
+            .attr("height", dimensions.height / 6)
+            .attr("x", "20")
+            .attr("y", "50");
+
+        const svgDefs = svgBar.append("defs");
+        const mainGradient = svgDefs.append("linearGradient").attr("id", "mainGradient");
+        mainGradient.append("stop").attr("stop-color", "red").attr("offset", "0");
+        mainGradient.append("stop").attr("stop-color", "white").attr("offset", "0.5");
+        mainGradient.append("stop").attr("stop-color", "blue").attr("offset", "1");
+        bar.attr("fill", "url(#mainGradient)");
+
+        svgBar.append("text")
+            .attr("transform", "translate(20, 45)")
+            .style("text-anchor", "middle")
+            .text("Per 0");
+
+        svgBar.append("text")
+            .attr("transform", "translate(" + dimensions.width * 0.25 + ", 45)")
+            .style("text-anchor", "middle")
+            .text("Per 25");
+
+        svgBar.append("text")
+            .attr("transform", "translate(" + dimensions.width * 0.75 + ", 45)")
+            .style("text-anchor", "middle")
+            .text("Per 75");
+
+        svgBar.append("text")
+            .attr("transform", "translate(" + dimensions.width + ", 45)")
+            .style("text-anchor", "middle")
+            .text("Per 100");
+
+        const percentile25 = svgBar
+            .append("rect")
+            .attr("width", 5)
+            .attr("height", dimensions.height / 6)
+            .attr("x", 0.25 * dimensions.width)
+            .attr("y", 50);
+
+        const percentile75 = svgBar
+            .append("rect")
+            .attr("width", 5)
+            .attr("height", dimensions.height / 6)
+            .attr("x", 0.75 * dimensions.width)
+            .attr("stroke", "black")
+            .attr("stroke-width", 2)
+            .attr("fill", "none")
+            .attr("y", 50);
+
+        const tooltip25 = svgBar.append("g").attr("class", "tooltip25").style("display", "none");
+
+        tooltip25
+            .append("rect")
+            .attr("width", 170)
+            .attr("height", 30)
+            .attr("x", 30)
+            .attr("y", 120)
+            .attr("fill", "lightGray");
+
+        tooltip25
+            .append("line")
+            .attr("stroke", "black")
+            .attr("x1", 100)
+            .attr("x2", 40)
+            .attr("y1", 80)
+            .attr("y2", 120);
+
+        tooltip25
+            .append("line")
+            .attr("stroke", "black")
+            .attr("x1", 105)
+            .attr("x2", 180)
+            .attr("y1", 80)
+            .attr("y2", 120);
+
+        tooltip25
+            .append("text")
+            .text("Quartile-25: " + localResolutionStats.data.metrics[1]?.["quartile 25"] + " Å")
```

Comment:
Not sure about this, just in case: I see that these two quartiles are shown on hover. But on the template of the ticket, it seems to always show all three values:

![image](https://user-images.githubusercontent.com/24643/177973390-5cf22102-e2c1-4f88-a9e7-6bc18c480e0d.png)

Reply:
this was a change @adrianq requested. i was asked to ignore the median for now until the client confirms and work with just the 25th and 75th quartile values on hover.